src/components/task/index.cjsx

```diff
@@ -1,11 +1,12 @@
-React = require 'react'
+React = require 'react/addons'
 BS = require 'react-bootstrap'
```

Comment:
is `addons` needed?  I don't see anywhere in the changes that you're using any of 'em.  I've probably just missed it though